stl/inc/ios

```diff
@@ -161,9 +161,9 @@ protected:
     __CLR_OR_THIS_CALL basic_ios() {}
 
 private:
-    _Mysb* _Mystrbuf; // pointer to stream buffer
-    _Myos* _Tiestr; // pointer to tied output stream
-    _Elem _Fillch; // the fill character
+    _Mysb* _Mystrbuf{}; // pointer to stream buffer
+    _Myos* _Tiestr{}; // pointer to tied output stream
+    _Elem _Fillch{}; // the fill character

```

Comment:
`_Elem` is potentially a user-provided type, but _incredibly_ unlikely to have issues with brace-init since the Standard makes the set of allowed types approximately implementation defined. I wouldn't mind `= _Elem()` here, but I won't complain either way.

Reply:
We have a bunch of occurrences of `_Elem{}` in `<xlocale>` and related headers, so I'd like to leave this as-is.